tests/integrationTests/tests/cpp_count_ts/__init__.py

```diff
@@ -0,0 +1,87 @@
+# Necessary imports. Provides library functions to ease writing tests.
+from lib import prebuild, testcase, SUBMITTY_INSTALL_DIR
+
+import subprocess
+import os
+import glob
+import shutil
+import traceback
+
+############################################################################
+# COPY THE ASSIGNMENT FROM THE SAMPLE ASSIGNMENTS DIRECTORIES
+
+SAMPLE_ASSIGNMENT_CONFIG = SUBMITTY_INSTALL_DIR + "/more_autograding_examples/cpp_count_ts/config"
+SAMPLE_SUBMISSIONS       = SUBMITTY_INSTALL_DIR + "/more_autograding_examples/cpp_count_ts/submissions"
+
+@prebuild
+def initialize(test):
+    try:
+        os.mkdir(os.path.join(test.testcase_path, "assignment_config"))
+    except OSError:
+        pass
+    try:
+        data_path = os.path.join(test.testcase_path, "data")
+        if os.path.isdir(data_path):
+            shutil.rmtree(data_path)
+        os.mkdir(data_path)
+    except OSError:
+        pass
+    subprocess.call(["cp",
+                     os.path.join(SAMPLE_ASSIGNMENT_CONFIG, "config.json"),
+                     os.path.join(test.testcase_path, "assignment_config")])
```

Comment:
It would be better to use `shutil.copyfile()` here.

Reply:
One clarification. I was basically copying the implementation of the existing integration tests and changing the `__init__.py ` to match this test case. I noticed none of the tests were using `shutil.copyfile()`. I am not sure if there is a specific reason for that. Should I go ahead and make this change?

Reply:
My understanding is that `subprocess.call` was used as something of a hack to mimic the CLI when prototyping the tests and then never removed.  I think it's probably best to not mimic the existing tests in this case.